deployment/bare_metal/setup.md

```diff
@@ -213,24 +211,11 @@ crontab -e
 This opens the crontab in edit-mode and insert the following entry:
 
 ```bash
-0 * * * * find /tmp -name "yarn--*" -cmin +60 -exec rm -r {} \; > /dev/null
-```
-
-### On `stage2`
-
-A daily job is necessary on `stage2` by setting the following job in the `crontab` for the `gradido` user.
-
-Run:
+0 * * * * find /tmp -name "yarn--*" -exec rm -r {} \; > /dev/null
 
-```bash
-crontab -e
 ```
 
-This opens the `crontab` in edit-mode and insert the following entry:
-
-```bash
-0 4 * * * find /tmp -name "yarn--*" -ctime +1 -exec rm -r {} \; > /dev/null
-```
+For production systems this is not need by default since the yarn output is deleted when `start.sh` is executed. If the service runs stable and does not restart frequently the yarn output to the tmp folder scales with the amount of services running.
```

Comment:
For production systems this is not needed by default since the yarn output is deleted when `start.sh` is executed. If the service runs stable and does not restart frequently, the yarn output to the tmp folder scales with the amount of services running.